src/lib/POWER_DETECT/devPDET.cpp

```diff
@@ -36,9 +37,27 @@ static int start()
     return DURATION_NEVER;
 }
 
+/**
+ * @brief Event callback for PDET device.
+ * 
+ * If the module is running in the "normal" mode, i.e. with the radio outputting RF,
+ * then set the initial duration to immediately call the 'timeout' function.
+ * Otherwise set the diration to never call the power detection/adjustment 'timeout' function.
+ * 
+ * @return int duration in ms to call the 'timeout' function
+ */
+static int event()
+{
+    if (GPIO_PIN_PA_PDET == UNDEF_PIN || connectionState > connectionState_e::MODE_STATES)
```

Comment:
Should MODE_STATES be replaced with noCrossfire?

Im trying to think what will happen if the handset comms takes a while to connect.  Will cause devPDET to never be called?

Reply:
The radio never starts until we reach the normal operating modes (i.e. < MODE_STATES).
Even in "airport" mode it skips noCrossfire and jumps to awaitingModelId which times out and moves to disconnected.

Reply:
Oh, even if it did take a while to connect, it would move state which would fire a new event.